virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,73 @@
+** Install xref:../virt/virtual_machines/virt-installing-qemu-guest-agent.adoc#virt-installing-virtio-drivers-installing-windows_virt-installing-qemu-guest-agent[VirtIO drivers] on the virtual machine.
+
+Connect to a virtual machine:
+
+* Connect to a virtual machine using xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-connecting-to-the-terminal_virt-accessing-vm-consoles[the terminal] or xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-vm-serial-console-web_virt-accessing-vm-consoles[the serial console] in the {product-title} web console.
+* Connect to a virtual machine using SSH:
+** xref:../virt/virtual_machines/vm_networking/virt-using-the-default-pod-network-with-virt.adoc#virt-using-the-default-pod-network-with-virt[Create a service] to expose a route for the virtual machine.
+** Use the service to connect to the virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vmi-ssh_virt-accessing-vm-consoles[SSH].
+
+Manage a virtual machine.
+
+* xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Stop, start, pause, and restart] the virtual machine.
+
+[id="virt-getting-started-next-steps"]
+== Next steps
+
+Network options:
+
+* xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc#virt-attaching-vm-multiple-networks[Configure a Linux bridge network], in addition to the default pod network.
+* Install the xref:../networking/hardware_networks/installing-sriov-operator.adoc#installing-sriov-operator[SR-IOV Operator], configure a xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[SR-IOV network device], and xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[attach a virtual machine to the SR-IOV network].
+
+Microsoft Windows virtual machine options:
+
+* xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Automate Windows virtual machine installations with `sysprep`].
+* Connect to a Microsoft Windows virtual machine with xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-rdp-console_virt-accessing-vm-consoles[RDP].
+
+Monitoring {VirtProductName}
+
+* Monitor virtualization resources, details, status, and top consumers on the  Review the xref:../virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc#virt-reviewing-virtualization-overview[*Virtualization Overview* page].
+* View high-level information about your virtual machines on the xref:../virt/logging_events_monitoring/virt-viewing-information-about-vm-workloads.adoc#virt-viewing-information-about-vm-workloads[*Virtual Machines* page.
+* View virtual machine xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs].
+
+[id="virt-getting-started-additional-resources"]
+== Additional resources
```

Comment:
Remember that we need use the ["role" tag ](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc)instead of an anchor ID for "additional resources". I wish I knew why!